### PR TITLE
Shortcut integrations update

### PR DESCRIPTION
**Pull request by cristopher-rodrigues**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,15 +4,16 @@ const moment = require("moment");
 
 const REPO = "rotabull";
 const OWNER = "rotabull";
-const CLUBHOUSE_BASE_URL = "https://app.clubhouse.io/rotabull/story/";
+const CLUBHOUSE_BASE_URL = "https://app.shortcut.com/rotabull/story/";
 const HEROKU_API_BASE_URL = "https://api.heroku.com";
 const GITHUB_API_BASE_URL = "https://api.github.com";
-const CLUBHOUSE_API_BASE_URL = "https://api.clubhouse.io/api/v3";
+const CLUBHOUSE_API_BASE_URL = "https://api.shortcut.com/api/v3";
```
Comment (Kysss) [inline index.js:10]: looking at the doc https://shortcut.com/api/rest/v3#We-changed-our-name, looks like the base url is 
**https://api.app.shortcut.com/api/v3/**

File: index.js [hunk 2]
```diff
@@ -41,8 +42,8 @@ async function run() {
           CHECK_STATUS_RETRIES,
           CHECK_STATUS_TIME_OUT
         );
-      case "update-clubhouse-workflow":
-        const storyIds = core.getInput("clubhouse-story-ids");
+      case "update-shortcut-workflow":
+        const storyIds = core.getInput("shortcut-story-ids");
```
Comment (Kysss) [inline index.js:46]: we also need to update these in `main.yml`

File: index.js [hunk 3]
```diff
@@ -90,15 +91,15 @@ function updateMultipleStories(stateId, storyIdsString){
 
   // storyIds needs to be an array of integers
   const storyIds = storyIdsString.split(",").map(id => parseInt(id));
-  
+
```
Comment (Kysss) [inline index.js:94]: header options  - e.g. line 66

Comment (cristopher-rodrigues) [reply index.js:94]: fixed https://github.com/rotabull/rotabull-release-helper/pull/20/commits/03f6d2e52f51cf62493f3590d13ad7787c4f3ccb

File: index.js (hunk 1 above)
Comment (cristopher-rodrigues) [reply index.js]: fixed https://github.com/rotabull/rotabull-release-helper/pull/20/commits/03f6d2e52f51cf62493f3590d13ad7787c4f3ccb

File: index.js (hunk 2 above)
Comment (cristopher-rodrigues) [reply index.js:46]: sure https://github.com/rotabull/rotabull/pull/2068/commits/51aca506c4fdcbf18503b2549ad176e95f278099 good catch 👨

File: __test__/index.test.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ const moment = require("moment");
 const axios = require("axios");
 const core = require("@actions/core");
 const main = require("../index.js");
+const { extractClubhouseNumberFromPRTitle, extractClubhouseNumberFromPRBody } = require("../index.js");
```
Comment (Kysss) [inline __test__/index.test.js:5]: think we can remove these - not used imports

**Review by Kysss — APPROVED** [review]

**Pull request merged**